### PR TITLE
SimpleMode: automatic public nodes discovering and switching

### PR DESCRIPTION
**Pull request by xiphon**
Split the PR into two commits, suggest to review them separately since the first one is pretty straightforward stripping `https://autonode.xmr.pm/` related code.

Will squash the commits once the PR is approved

### CODE REVIEW

**Comment by selsta** [comment]
https://github.com/monero-project/monero-gui/blob/3a9f0e11003168a13c3648d02c193402e67e95b2/js/Utils.js#L97 can be deleted too.

**Comment by xiphon** [comment]
> https://github.com/monero-project/monero-gui/blob/3a9f0e11003168a13c3648d02c193402e67e95b2/js/Utils.js#L97
> can be deleted too.

Yep, but decided to keep it cause it's pretty common helper routine.

**Comment by xiphon** [comment]
Updated. Dropped unused `randomChoice` routine.

File: main.qml [hunk 1]
```diff
@@ -2034,29 +2033,21 @@ ApplicationWindow {
...
-        // disconnected longer than 5 seconds?
-        if(disconnected && disconnectedEpoch > 0 && (Utils.epoch() - disconnectedEpoch) >= 5){
-            // for bootstrap mode, first wait until daemon is killed
-            if(appWindow.walletMode === 1 && appWindow.daemonRunning) {
+        if (haveToChangeDaemonNoSyncFlag()) {
+            appWindow.stopDaemon();
+        }
+
+        // disconnected longer than N seconds?
+        if(disconnected && disconnectedEpoch > 0 && (Utils.epoch() - disconnectedEpoch) >= 30){
```
Comment (selsta) [inline main.qml]: It takes 30 seconds now after opening the GUI (simple mode) until the daemon starts. Is this intended?

**Comment by selsta** [comment]
Closing the wallet in simple mode shows this, not sure if that’s related to this PR:

`2019-09-05 20:37:25.629	W qrc:/main.qml:682: TypeError: Type error`

File: main.qml (hunk 1 above)
Comment (xiphon) [reply main.qml]: fixed

**Comment by xiphon** [comment]
> Closing the wallet in simple mode shows this, not sure if that’s related to this PR:
> 
> `2019-09-05 20:37:25.629 W qrc:/main.qml:682: TypeError: Type error`


Could you provide reproduction steps?

**Comment by selsta** [comment]
I would even reduce the initial delay time from 5 to like 2 seconds.

> Could you provide reproduction steps?

It shows up every time I close the wallet in simple mode.

**Comment by selsta** [comment]
Were you able to reproduce the `2019-09-09 15:16:41.017	W qrc:/main.qml:682: TypeError: Type error` issue? PR looks good otherwise.

**Comment by xiphon** [comment]
> Were you able to reproduce the `2019-09-09 15:16:41.017 W qrc:/main.qml:682: TypeError: Type error` issue? PR looks good otherwise.

Yep, the issue is not related to the PR

**Comment by xiphon** [comment]
Squashed

**Pull request merged**